RELEASE/scripts/autoscend/auto_equipment.ash

```diff
@@ -73,10 +73,22 @@ boolean autoOutfit(string toWear)
 	{
 		// yes I could use +outfit instead here but this makes it simpler to avoid failed maximize calls
 		auto_debug_print('Adding outfit "' + toWear + '" to maximizer statement', "gold");
+
+		// Accessory items from outfits we commonly wear
+		boolean[item] CommonOutfitAccessories = $items[eXtreme mittens, bejeweled pledge pin, round purple sunglasses];
```

Comment:
i see outfit references to swashbuckling getup still, though it might not be using autoOutfit to equip. Still probably worth adding stuffed shoulder parrot here for posterity. Also see https://kol.coldfront.net/thekolwiki/index.php/Vile_Vagrant_Vestments referenced once, which is weird but i guess we could add Oscus's pelt too. i didnt look to see where or why thats used.

Reply:
Found where Vile Vestments were equipped. It was using outfit() rather than autoOutfit(), I changed that.

It seems we look at the Pirate Outfit but never actually equip it. I still added the Parrot just in case that changes in the future.